### DISCUSSION TITLE
L1, L2 and Roll-ups

### DISCUSSION THREAD

**Discussion by rijima-25** [Cyfrin/foundry-full-course-cu#2962] [Q&A]
I just finished the Blockchain basics course on Cyfrin, and I must say, it was really insightful. Lots of new things to know. But I'm still a bit confused about the L1, L2 and roll ups. Can someone explain it to me like I'm a child? 😂 Roll-ups and L2s aren't the same thing, right? How do they differ? 

Secondly, I'm having issues following through with setting up my metamask wallet, the tutorial was done using a chrome extension of Metamask and I'm using the app - which has a pretty different and almost very simplified UI, how do I navigate it?

**Comment by heheboii11** [comment]
L1 are the base blockchains such as Ethereum, Aptos, Solana. For getting understanding between L2 and rollups please refer to this blog which clearly explains the difference between L2 and rollups 
https://medium.com/@Ground_Zero/ethereum-l2-solutions-vs-rollups-understanding-the-difference-a93f5108bac5
Coming to difficulties to setup the metamask, please use the extension version in your laptop as it is very difficult for to handle the development part with mobile app

**Reply by EngrPips** [reply]
Great heads-up.

**Reply by rijima-25** [reply]
Thank you Ramireddy Karthik Reddy. 🙏🏽
